packages/mui-material/test/integration/Select.test.js

```diff
@@ -101,18 +84,6 @@ describe('<Select> integration', () => {
   });
 
   describe('with label', () => {
-    /**
```

Comment:
Should we call `clock.withFakeTimers()` only when we invoke `clock.tick(VALUE)` with `VALUE` other than zero?

Reply:
you can't call `clock.tick` outside of fake timers. This would only work if we make it async and resolve it via `setTimeout`. 

Whether you want to use fake timers it not necessarily tied to calling `clock.tick` (which you should only call with a number. whether it's zero or N does not matter. Zero just means the shortest possible delay. Depending on the runtime this might be equivalent to 10 though). Using fake timers might be useful if you don't want any timers to run at all in your test. 

Generally, fake timers make your test more predictable. With real timers you might have different updates depending on how fast your test is.

To be clear: `withFakeTimers` is just a replacement for 

```js
  /**
   * @type {ReturnType<typeof useFakeTimers>}
   */
  let clock;
  beforeEach(() => {
    clock = useFakeTimers();
  });

  afterEach(() => {
    clock.restore();
  });
```

Reply:
Yes, it's much more clear, thanks for the clarification.

> To be clear: withFakeTimers is just a replacement for

Then, shouldn't we add it here too? What confused me is why we haven't added it here.

Reply:
Ah, good catch. I put the whole test into fake timers mode instead of just this block. Will fix.

Reply:
Now I remember why: The tests had only two `describe` blocks that each set up their own clock. Now we run with the same clock for both i.e. we just hoisted this setup to before `createRenderer` first and then applied the new pattern.

Reply:
Ok, but still we should call `clock.withFakeTimers()` on line 12, no? After: 
```
const { clock, render } = createRenderer({ clock: 'fake' });
```

Reply:
The clock is already fake due to `{ clock: 'fake' }`. Why do you expect to be required to call `withFakeTimers` again?

Reply:
Ahhh, ok I see the connection now, I haven't noticed that sometimes we use the `clock: 'fake'` inside the `createRenderer`

Reply:
Hmm, not sure how the API could be structured to avoid this confusion.